reactor-extra/src/main/java/reactor/bool/BoolUtils.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.bool;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * Functions to help compose results with {@link Mono}-wrapped {@literal boolean} values.
+ *
+ * @author Greg Turnquist
+ */
+public final class BoolUtils {
```

Comment:
maybe rename to `BooleanUtils` for consistency with `Mono<Boolean>`?

Reply:
Actually I was going for that, but discovered that `reactor.boolean` wasn't a valid package name due to `boolean` being a reserved word.

Either way, I can go for.

Reply:
`bool` in the package is fine by me, the class name feels weird though